src/filters/FullPoseEstimator.cpp

```diff
@@ -23,7 +24,7 @@ constexpr int HEADING_IDX = 1;
 state_t stateFunc(const DiffDriveKinematics& kinematics, double dt, const state_t& x,
 				  const action_t& u, const action_t& w) {
 	action_t input = u + w; // noise is applied to the input vector
-	wheelvel_t wheelVel{input(0), input(1)};
+	kinematics::wheelvel_t wheelVel{input(0), input(1)};
```

Comment:
Now that you specified `use namespace kinematics` on line 7, the explicit namespace specifier is unneeded